File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao05.kt
fun main() {
    println("Digite um número inteiro: ")
    var numero_1 = readLine()!!.toInt()
    
    println("Digite um número inteiro: ")
    var numero_2 = readLine()!!.toInt()
    
    var soma = 0

    var maior = if(numero_1 < numero_2) numero_2 else numero_1
    var menor = if(numero_1 > numero_2) numero_2 else numero_1

    for (cont in menor .. maior) {
        soma += cont
        println(cont)
    }

    print("A soma desses números é: $soma)
}

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao05.js
let numero_1 = parseInt(prompt("Digite um número inteiro: "))
let numero_2 = parseInt(prompt("Digite um número: "))
let maior = numero_1 < numero_2 ? numero_2 : numero_1
let menor = numero_1 > numero_2 ? numero_2 : numero_1

for (var cont = menor; cont <= maior; cont++){
  console.log(cont)
}

console.log("A soma desses números é: ${soma}")